.github/workflows/clippy.yml

```diff
@@ -0,0 +1,8 @@
+name: Clippy
+on: [push]
+jobs:
+  clippy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
This is the latest version of this action; other repositories that use older versions of it will probably have warnings about using a `deprecated` version of `Node.js`.